docs/Code-Reviews.md

```diff
@@ -0,0 +1,161 @@
+
+**Approvals**
+GitHub supports the concept of 'approvals'. Our repos generally require **two** approving reviews before a change can be merged on GitHub. Approvals should usually come from people who were not directly involved in implementing the change.
+
+
+- Anyone can approve a PR, regardless of seniority or experience
+- Approving a PR does not mean that it cannot be improved upon in some way
+- Similarly, approving a PR does not mean you will be held personally responsible for it
+    - If problems are found later, we'll fix them together, work out what went wrong and try to do better next time.
+## Our approach to reviewing code
+
+Code reviews are not an exercise in scoring points or proving that the reviewer knows better than the author - they are about trying to help each other to make the best quality work we can. Ideally, both author and reviewer come away from a code review having both made a positive contribution to the result and learned something along the way.
+
+Software development is a collaborative process. We own the whole codebase collectively across our teams. Nobody has one special responsibility or authortity over any one part of it. Feedback on code should never feel personal but should always be directed at improving our solutions and how we work together.
+
+Whatever problems we may find as we explore and discuss code, we assume that everyone is (and was) acting with best intentions, based on what they knew at the time and under circumstances that are often lost to us. Instead of complaining, we propose concrete steps we can take to make things better.
+
+## How long should it take?
+
+Although we value code review highly for the reasons given above, we also recognise that we have limited time and resources and a long list of work to complete. Time spent on code review should be kept in proportion with other stages of the development life cycle including testing and accessibility and UX review.
+
+The amount of time spent on code review will vary from task to task, but as a rule of thumb, if a PR has been at the code review stage for longer than it was in the initial implementation, it has been there too long. Both authors and reviewers have a shared responsibility to ensure that code review is completed in a timely fashion.
+
+We strive for quality but we do not aim for perfection ('perfect' code does not exist anyway!).
+
+## Reducing the effort needed in code review
+
+Pair programing and swarming/mobbing during the implementation phase are approaches that can reduce the time spent on lengthy in-depth code reviews later on, because more of the necessary communication has already happened. More people contributing to the solution may help to catch some types of issues before code review. Seeking alignment through early collaboration can therefore minimise the back-and-forth sometimes seen in code reviews.
```

Comment:
```suggestion
Pair programming and swarming/mobbing during the implementation phase are approaches that can reduce the time spent on lengthy in-depth code reviews later on, because more of the necessary communication has already happened. More people contributing to the solution may help to catch some types of issues before code review. Seeking alignment through early collaboration can therefore minimise the back-and-forth sometimes seen in code reviews.
```